recipes/hdrhistogram-c/all/patches/0001-add-conan-support.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 72261da..894fd18 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -2,6 +2,9 @@ cmake_minimum_required(VERSION 3.5)
+ 
+ project(hdr_histogram VERSION 0.10.0 LANGUAGES C)
+ 
++include(${PROJECT_BINARY_DIR}/../conanbuildinfo.cmake)
++conan_basic_setup()
++
+ include(GNUInstallDirs)
+ include(CMakePackageConfigHelpers)
+ 
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 9dbb311..5bd6ca4 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -1,10 +1,3 @@
+-find_package(ZLIB REQUIRED)
+-find_package(Threads REQUIRED)
+-
+-include(CheckLibraryExists)
```

Comment:
It should work normal without this patch. All system libs should listed according in cpp_info.system_libs

Reply:
Done, cf. 2fa63c9